x-pack/plugins/observability_solution/slo/public/pages/slos/components/common/quick_filters.scss

```diff
@@ -0,0 +1,40 @@
+.controlsWrapper {
```

Comment:
can we please avoid these custom styles here? isn't it possible to fix this via a prop?

Reply:
It is unfortunately not possible to customize via a prop, no.